sql/20201027-routed.sql

```diff
@@ -0,0 +1,38 @@
+ALTER TABLE threads
+ADD COLUMN routed boolean;
+
+
+--- populate routed threads
+WITH threads_with_rank AS (
+    SELECT *, row_number() OVER (PARTITION BY user_id, is_demo ORDER BY updated_at DESC) AS rn
+    FROM threads
+)
+UPDATE threads
+SET routed='t'
```

Comment:
can you update this to `routed = true` to be consistent with your other migrations?